src/Chirp.Infrastructure/CheepRepository.cs

```diff
@@ -29,6 +29,7 @@ public async Task AddCheep(int authorId, string text)
             {
                 _db.Add(new Cheep
                 {
+                    CheepId = new Guid(),
```

Comment:
I think that this is not the way you use Guid. You should use Guid.NewGuid() instead of new Guid().